examples/flutter-ota/pubspec.yaml

```diff
@@ -34,9 +34,7 @@ dependencies:
   flutter_localizations:        # Add this line
     sdk: flutter                # Add this line   
   intl: ^0.17.0                 # Add this line 
-  lokalise_flutter_sdk: ^0.4.0  # Add this line
-  #lokalise_flutter_sdk: 
-  #  path: /Users/jorgevegas/source/lokalise-flutter-sdk
+  lokalise_flutter_sdk: ^0.5.0  # Add this line
```

Comment:
Using the latest version

Reply:
btw, I will update it as today we have just released a new version with a bug fix